pkg/flowaggregator/flowaggregator.go

```diff
@@ -65,14 +67,62 @@ var (
 		"egressNetworkPolicyName",
 		"egressNetworkPolicyNamespace",
 	}
-	aggregationFields = []string{
+
+	nonStatsElementList = []string{
+		"flowEndSeconds",
+	}
+	statsElementList = []string{
+		"octetDeltaCount",
+		"octetTotalCount",
+		"packetDeltaCount",
+		"packetTotalCount",
+		"reverseOctetDeltaCount",
+		"reverseOctetTotalCount",
+		"reversePacketDeltaCount",
+		"reversePacketTotalCount",
+	}
+	antreaSourceStatsElementList = []string{
+		"octetDeltaCountFromSourceNode",
+		"octetTotalCountFromSourceNode",
+		"packetDeltaCountFromSourceNode",
+		"packetTotalCountFromSourceNode",
+		"reverseOctetDeltaCountFromSourceNode",
+		"reverseOctetTotalCountFromSourceNode",
+		"reversePacketDeltaCountFromSourceNode",
+		"reversePacketTotalCountFromSourceNode",
+	}
+	antreaDestinationStatsElementList = []string{
+		"octetDeltaCountFromDestinationNode",
+		"octetTotalCountFromDestinationNode",
+		"packetDeltaCountFromDestinationNode",
+		"packetTotalCountFromDestinationNode",
+		"reverseOctetDeltaCountFromDestinationNode",
+		"reverseOctetTotalCountFromDestinationNode",
+		"reversePacketDeltaCountFromDestinationNode",
+		"reversePacketTotalCountFromDestinationNode",
+	}
+	aggregationElements = &ipfixintermediate.AggregationElements{
+		NonStatsElements:                   nonStatsElementList,
+		StatsElements:                      statsElementList,
+		AggregatedSourceStatsElements:      antreaSourceStatsElementList,
+		AggregatedDestinationStatsElements: antreaDestinationStatsElementList,
+	}
+
+	correlateFields = []string{
+		"sourcePodName",
+		"sourcePodNamespace",
+		"sourceNodeName",
 		"destinationPodName",
 		"destinationPodNamespace",
 		"destinationNodeName",
 		"destinationClusterIPv4",
+		"destinationClusterIPv6",
+		"destinationServicePort",
```

Comment:
This will not work because that field is not added in Flow Exporter.
It is needed. This will fix it: https://github.com/vmware-tanzu/antrea/pull/1641